assets/js/src/wizard/welcome_wizard_controller.jsx

```diff
@@ -135,21 +134,6 @@ const WelcomeWizardStepsController = (props) => {
           </WelcomeWizardStepLayout>
         ) : null}
 
-      { stepName === 'WelcomeWizardWooCommerceStep'
-        ? (
-          <WelcomeWizardStepLayout
-            step={step}
-            stepsCount={stepsCount}
-            illustrationUrl={window.wizard_woocommerce_illustration_url}
-          >
-            <WelcomeWizardWooCommerceStep
-              next={() => redirect(step)}
-              screenshot_src={window.wizard_woocommerce_box_url}
```

Comment:
These two URL definitions should also be removed https://github.com/mailpoet/mailpoet/blob/master/views/welcome_wizard.html#L9-L10  (and also `is_woocommerce_active` 3 lines below)